exporter/prometheusexporter/prometheus.go

```diff
@@ -70,6 +85,10 @@ func (pe *prometheusExporter) Start(_ context.Context, _ component.Host) error {
 		return err
 	}
 
+	if err := view.Register(metricViews()...); err != nil {
+		log.Fatalf(err.Error())
+	}
```

Comment:
This will cause an error if multiple prometheus exporters are added to a collector.

Reply:
moved to init